### ISSUE TITLE
rename master to main

### ISSUE THREAD

**Issue by rettinghaus** [music-encoding/sibmei#169]
Anyone against this?

**Comment by th-we** [comment]
I can't find the issue right now, but we discussed abandoning master in favor of branch names `mei3`, `mei4` etc., depending on the exported MEI version. I started working on text export problems before making the MEI 4 release and introduce the new branching schema – I just didn't get around to finishing it yet. Thanks for nudging!

**Comment by annplaksin** [comment]
@th-we Do you mean that comment? https://github.com/music-encoding/sibmei/pull/159#issuecomment-754582217

**Comment by ahankinson** [comment]
Do we intend to support mei3 in the long term? If so, then we can have separate branches, but the trajectories of the branches would probably diverge to the point that they would effectively become independent forks. At this point, it might make sense to actually fork the repo to a dedicated project for legacy formats, instead of trying to maintain divergent branches on the same repo.

I tend to think of branches as always, ultimately, heading back to a single 'flow' of code. Feature to develop, develop to main, etc. Of course you can have 'forked branches' (feature `x` to develop branch `a`, feature `x` to develop branch `b`, develop branch (`a, b`) to main (`a, b`)) but this gets messy.

In other words, if we have `mei3` and `mei4` would we need `develop-mei3` and `develop-mei4`?

**Comment by th-we** [comment]
@ahankinson I'm not eager to support MEI 3 in the long term, but if it's necessary for someone or some project to get an occasional bugfix, then it's good to have a dedicated branch (instead of just a tag).

Good points about the develop branch(es). I vote for keeping `develop` (which always belongs to the latest `meiX` branch) and, if necessary, create a `develop-mei3` branch. That way, we can work continuously on the `develop` branch, also while making changes to transition from one MEI version to the next.

**Comment by ahankinson** [comment]
Would the latest `meiX` branch effectively equal what we would expect on `main`? If so, then I would vote for a name like `main`. We could have dedicated legacy branches, like `mei3`, but the challenge would be whether `mei4` is the latest branch or not... what happens when we need `mei5`? 

So I think I'm OK with the idea of an `mei3` branch for the legacy code, but I'm not so convinced on `mei4` for the most recent name.

**Comment by th-we** [comment]
So can I go forward with the following steps:
* rename `master` to `main`
* merge `develop` into `master`
* create a new branch `mei3` at 73cf79e (v2.0.3)
* Release Sibmei 4 from `main`

**Comment by rettinghaus** [comment]
@th-we merge #159 in `mei3` and have v2.3.1?

**Comment by th-we** [comment]
@rettinghaus O.K., so I'll prepare two releases. The release tags should be v2.0.4 and v4.0.0.

**Comment by ahankinson** [comment]
> merge `develop` into `master`

Do you mean `main`? Or did I miss something?

**Comment by th-we** [comment]
Yes, that was mistyped.

**Comment by th-we** [comment]
I've done all the branch juggling, set the v4.0.0 tag and created a release draft. Could a second pair of eyes check if the branches are correct and the release draft makes sense? Please feel free to do corrections to the draft and publish it when it's O.K.

**Comment by ahankinson** [comment]
I've had a look.

 - Should we merge #159 into the `mei3` branch?
 - Should we also do a release on the `mei3` branch?

**Comment by th-we** [comment]
I merged that branch locally, but had weird problems running the tests for mei3 on the latest Sibelius Ultimate. I would like to sort that out before releasing, but likely not before next weekend.

However, I would like to publish the release for Sibmei 4, if nothing speaks against it.

**Comment by ahankinson** [comment]
Nothing from me for v.4. Any idea about my earlier questions?

**Comment by th-we** [comment]
You mean #159? That's the one I meant when I said "I merged that branch locally". Works as intended and I'll merge it.

**Comment by th-we** [comment]
I published the 4.0.0 release now. I'll leave this issue open until I sorted out the mei3 testing.

**Comment by rettinghaus** [comment]
@th-we any updates on this?

**Comment by th-we** [comment]
Branches are renamed, so closing this.
